src/command-line/config.js

```diff
@@ -6,9 +6,12 @@ program
 	.command("config")
 	.description("Edit config: " + Helper.CONFIG_PATH)
 	.action(function() {
-		child.spawn(
-			process.env.EDITOR || "vi",
+		var child_spawn = child.spawn(
+			process.env.EDITOR || "vi" || "nano",
```

Comment:
Your ` || "nano"` addition doesn't make much sense, as the `vi` fallback is for the `EDITOR` being `undefined`.